medkumo.js

```diff
@@ -367,6 +485,13 @@
         if (dobValue == null || dobValue == "") {
             result['dob'] = 'This field is required';
```

Comment:
move these message to Config. 

Ex: 
Config.message.required = 'This field is required';
Config.message.dob = 'The date of birth Invalid';